modules/server/src/download/index.js

```diff
@@ -9,9 +9,10 @@ import getAllData from '../utils/getAllData';
 import dataToTSV from '../utils/dataToTSV';
 
 export default function({ projectId }) {
-  function makeTSV(args) {
+  function makeTSV(args, mock) {
```

Comment:
What are "args" here? the actual object data?

Reply:
yep, so if you make a request thats body contains `files: [file1]` args here would be file1.

Reply:
I could add mock to the file options, that would allow individual mocking as well.